react/features/base/participants/functions.ts

```diff
@@ -349,15 +373,16 @@ export function getParticipantCountWithFake(stateful: IStateful) {
  * @returns {string}
  */
 export function getParticipantDisplayName(stateful: IStateful, id: string): string {
-    const participant = getParticipantById(stateful, id);
+    const state = toState(stateful);
+    const participant = getParticipantById(state, id);
     const {
         defaultLocalDisplayName,
         defaultRemoteDisplayName
-    } = toState(stateful)['features/base/config'];
+    } = state['features/base/config'];
 
     if (participant) {
-        if (participant.isVirtualScreenshareParticipant) {
-            return getScreenshareParticipantDisplayName(stateful, id);
+        if (isScreenShareParticipant(participant)) {
```

Comment:
not sure if intended, but the original check was only acting when fake participant was virtual screen share, while the refactored one will return true for both local and virtual

Reply:
The local screenshare was marked as both virtual screenshare participant and local (search for isLocalScreenShare), so I had to check for both where isVirtualScreenshareParticipant was used as they're separate types now, wdyt?

Reply:
I think both can be considered virtual, so I renamed the non-local one to remote